azurerm/internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -437,6 +437,12 @@ func resourceArmKubernetesCluster() *schema.Resource {
 				ConflictsWith: []string{"private_link_enabled"},
 			},
 
+			"private_dns_zone_id": {
```

Comment:
could we [add this new field to the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/kubernetes_cluster.html.markdown#L101)?

Reply:
Yeah, certainly, I wasn't done with this one yet, since I didn't know how long UserAssigned identities will take to merge. I'll finalize this tonight or tomorrow.